demisto_sdk/commands/common/hook_validations/dashboard.py

```diff
@@ -12,14 +12,15 @@ def get_widgets_from_dashboard(dashboard):
             widgets = [item.get('widget') for item in layout_of_dashboard]
         return widgets
 
-    def is_valid_dashboard(self):
+    def is_valid_dashboard(self, validate_rn=True):
         # type: () -> bool
         """Check whether the dashboard is valid or not.
 
         Returns:
             bool. Whether the dashboard is valid or not
         """
         is_dashboard_valid = [
+            super(DashboardValidator, self).is_valid_file(validate_rn),
```

Comment:
in python3 you don't need those args
```suggestion
            super().is_valid_file(validate_rn),
```

Reply:
@yaakovi seems like it is needed.
```
        return all(
            [
>               super.is_valid_file(validate_rn),
                self.is_valid_name(),
                self.is_valid_type(),
                self.is_valid_group(),
                self.is_valid_content_flag(),
                self.is_valid_system_flag(),
                self.is_valid_cliname(),
                self.is_valid_version(),
                self.is_current_valid_from_version(),
                self.is_valid_required()
            ]
        )
E       AttributeError: type object 'super' has no attribute 'is_valid_file'

demisto_sdk/commands/common/hook_validations/incident_field.py:182: AttributeError
```

Reply:
same error occurs in the `dashboard.py` upon change.

Reply:
`super()` not `super`